lib/babylon-to-espree/convertAST.js

```diff
@@ -47,26 +47,6 @@ const astTransformVisitor = {
   exit(path) {
     const node = path.node;
 
-    if (path.isJSXText()) {
-      node.type = "Literal";
-    }
-
-    if (
-      path.isRestElement() &&
-      path.parent &&
-      path.parent.type === "ObjectPattern"
-    ) {
-      node.type = "ExperimentalRestProperty";
```

Comment:
Is`Array{Pattern|Expression}` already taken care of?

Reply:
Yes.

https://github.com/babel/babel-eslint/blob/2540427c555cf707ced5b71b1717ac401fbcf3d0/test/specs/babel-eslint.js#L485-L495

Reply:
Great, thanks - makes sense!